src/duplicate-search/__tests__/duplicate-search-controls.test.tsx

```diff
@@ -513,5 +570,23 @@ describe( '[DuplicateSearchControls]', () => {
 				).toHaveFocus()
 			);
 		} );
+
+		test( 'Selecting a sort option records event', async () => {
+			const { monitoringClient, user } = setup( <DuplicateSearchControls /> );
+
+			await user.click( screen.getByRole( 'combobox', { name: 'Sort results by…' } ) );
+			await user.click( screen.getByRole( 'option', { name: 'Date added' } ) );
+
+			expect( monitoringClient.analytics.recordEvent ).toHaveBeenCalledWith( 'sort_select', {
+				sortOption: 'date-created',
```

Comment:
**nitpick (non-blocking):**
I noticed that the event name is `date-created`, whereas the filter in the UI is `Date added`. This made me realize that we've been a bit inconsistent with our naming of this field because even the duplicate search result row uses `Opened 4 days ago` lol.

I feel like we should probably pick one and keep it the same throughout duplicate search 😆 For example, the filter and event name could be `date created` and the results row could display `Created 4 days ago`. 

Karen and @dpasque, what are your thoughts on this?

Reply:
> I feel like we should probably pick one and keep it the same throughout duplicate search 😆 For example, the filter and event name could be date created and the results row could display Created 4 days ago.

That's a good point! Since `date created` is used mostly everywhere, we can update the text in the dropdown from `Date added` to `Date created`.

Reply:
Created a separate PR for this change: https://github.com/Automattic/bugomattic/pull/110